packages/react/src/tooltip-content/TooltipContent.css.ts

```diff
@@ -0,0 +1,14 @@
+import { recipe } from "../vanilla-extract";
+
+export const content = recipe({
+  base: [
+    {
+      bg: "bg.neutral.inverse",
+      color: "white",
+      px: "12",
+      py: "8",
+      rounded: "lg",
+      z: "popover",
```

Comment:
let's add a new zIndex for tooltips and use a higher value - like `10000`
```suggestion
      z: "tooltip",
```

and in `packages/globals/src/tokens/zIndex.ts` add an entry for `tooltip: "10000"`